src/integrationTest/resources/xmlrequest/ClaimDefences.xml

```diff
@@ -0,0 +1,16 @@
+  <sub:submitQueryRequest xmlns:sub="http://ws.sdt.moj.gov.uk/2013/sdt/SubmitQueryRequestSchema"
+                              xmlns:quer="http://ws.sdt.moj.gov.uk/2013/mcol/QuerySchema">
+      <sub:header>
+        <sub:sdtCustomerId>12345678</sub:sdtCustomerId>
+        <sub:targetApplicationId>mcol</sub:targetApplicationId>
+        <sub:queryReference>optional</sub:queryReference>
+      </sub:header>
+      <sub:criteria>
+        <sub:criterion criteriaType="mcolDefenceCriteria">
+          <sub:mcolDefenceCriteria>
+            <quer:fromDate>2009-12-01</quer:fromDate>
+            <quer:toDate>2009-12-02</quer:toDate>
```

Comment:
To accurately reflect an actual query request, the from and to dates need time components, e.g. `2009-12-01T00:00:00`.

Reply:
done